app/script/auth/page/Root.js

```diff
@@ -18,18 +18,49 @@
  */
 
 import React from 'react';
-import {indexStrings} from '../../strings';
-import {injectIntl} from 'react-intl';
 import {connect} from 'react-redux';
+import {Logo} from '@wireapp/react-ui-kit/Identity';
+import {Small, H3, Link} from '@wireapp/react-ui-kit/Text';
+import {StyledApp, Content, Columns, Column, ContainerXS} from '@wireapp/react-ui-kit/Layout';
 
-const Root = ({name, intl: {formatMessage: _}}) => (
-  <h1>
-    {_(indexStrings.hello)} {name}!
-  </h1>
+const Root = ({name}) => (
+  <StyledApp>
+    <Content>
+      <ContainerXS centerText verticalCenter>
+        <br />
+        <Logo id="wire-logo" />
+        <br />
+        Secure messaging for everyone
```

Comment:
I guess this will be translated and stuff.. but until then maybe `{'Secure messaging for everyone'}`...?